apps/api/src/app/controllers/__tests__/__snapshots__/userManagementQueries.e2e.spec.ts.snap

```diff
@@ -1207,6 +1207,72 @@ exports[`User Management Queries when the system user issuing the command is aut
       ],
       "username": "cool-james",
     },
+    {
+      "actions": [
+        {
+          "description": "Grant an additional user role to the user",
+          "form": {
+            "fields": [
+              {
+                "constraints": [
+                  {
+                    "message": "Must be a valid \${propertyLabel}",
+                    "name": "IS_ENUM",
+                  },
+                ],
+                "description": "the role that will be granted to this user",
+                "label": "User Role",
+                "name": "role",
+                "options": [
+                  {
+                    "display": "admin",
+                    "value": "projectAdmin",
+                  },
+                  {
+                    "display": "viewer",
+                    "value": "viewer",
+                  },
+                  {
+                    "display": "COSCRAD admin",
+                    "value": "superAdmin",
+                  },
+                ],
+                "type": "STATIC_SELECT",
+              },
+            ],
+            "prepopulatedFields": {
+              "aggregateCompositeIdentifier": {
+                "id": "9b1deb4d-3b7d-4bad-9bdd-2b0d7b110002",
+                "type": "user",
+              },
+            },
+          },
+          "label": "Grant User Role",
+          "type": "GRANT_USER_ROLE",
+        },
+      ],
+      "id": "9b1deb4d-3b7d-4bad-9bdd-2b0d7b110002",
+      "name": {
+        "items": [
+          {
+            "languageCode": "eng",
+            "role": "original",
+            "text": "cscrad-cypress-user",
```

Comment:
I assume no "o" is consistent with auth0

Reply:
It has nothing to do with auth0. As I said when I explained this to you recently, only the  **auth0 ID** is the link to our database, which is the source of truth for everything else. That said, thanks for pointing out this typo, which I have fixed!